contracts/Bridge.sol

```diff
@@ -146,14 +159,14 @@ contract Bridge is Pausable, AccessControl, SafeMath {
     }
 
     /**
-        @notice Removes admin role from {msg.sender} and grants it to {newAdmin}.
+        @notice Removes admin role from {_msgSender()} and grants it to {newAdmin}.
         @notice Only callable by an address that currently has the admin role.
         @param newAdmin Address that admin role will be granted to.
      */
     function renounceAdmin(address newAdmin) external onlyAdmin {
-        require(msg.sender != newAdmin, 'Cannot renounce oneself');
+        require(_msgSender() != newAdmin, 'Cannot renounce oneself');
```

Comment:
Because _msgSender() reads from storage, we should avoid using it more than once per external function. Read it into a local var, then use that.
```
address sender = _msgSender();
```

Reply:
make sense :)